en/05_Contributing/06_Documentation.md

```diff
@@ -1,141 +1,250 @@
+If you haven't contributed documentation in the past, you will need to fork the repository. If you're editing online, GitHub will guide you through that process before you can edit the documentation.
 
-## Source control
+### Simple changes {#editing-online-simple}
 
-Contributing documentation is the same process as providing any other patch (see [Contributing code](code)).
+The easiest way of editing any documentation is by clicking the "Edit on GitHub" button at the bottom of the page you want to edit. This works great if all of the changes you want to make are on a single page.
 
-If you are fixing incorrect or incomplete information for the current major version, you should create a PR that targets the branch for the latest stable release (e.g. `4.11`).
+Before you start making your changes, take note of what branch you are editing. You will need this when you create the pull request.
 
-If you are adding documentation for functionality that has not yet been released, you should target the most recent _major_ branch (e.g. `4`)
+![note the branch you're editing](../_images/contributing/docs-check-the-branch.png)
 
-## What to write
+After editing the documentation, click "Commit changes", then describe your changes in the "commit summary" and "extended description" fields below.
 
-See [what to write (jacobian.org)](http://jacobian.org/writing/great-documentation/what-to-write/) for an excellent introduction to the different types of documentation. Also see [producing OSS: "documentation"](http://producingoss.com/en/getting-started.html#documentation) for good rules of thumb 
-for documenting open source software.
+![follow the guidelines to add a descriptive commit message](../_images/contributing/docs-commit-message.png)
 
-## Structure
+Click "Propose changes" when you are ready to commit your changes. This saves them to the forked repository on your profile.
 
-* Keep documentation lines shorter than 120 characters.
-* Don't duplicate: Search for existing places to put your documentation. Do you really require a new page, or just a new paragraph of text somewhere?
-* Use PHPDoc in source code: Leave low level technical documentation to code comments within PHP, in [PHPDoc](http://en.wikipedia.org/wiki/PHPDoc) format. 
-* API and developer guides are two forms of source code documentation that complement each other.
-* API documentation should provide context, ie, the "bigger picture", by referring to developer guides inside your PHPDoc.
-* Make your documentation easy to find: Documentation is useful only when it is interlinked so please make sure your contribution doesn't become an inaccessible island. At the very least, put a link to your index page in the same folder. A link to your page can also appear
-as "related content" on other resource (e.g. `/tutorials/site_search` might link to `/developer_guides/forms/introduction`).
```

Comment:
I've moved any relevant points to the style section.
Most of this is outdated IA stuff. We're going to have to redo our IA eventually as it's currently a mess and is really hard to find relevant documentation (in part because some of these suggestions simply didn't scale well).
In the meantime, I can't see anyone adding substantial enough docs to need this part of the guide.